contracts/DelegateContribute.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.23;
 
-interface IWVEOLAS {
+interface IVEOLAS {
```

Comment:
`veOLAS` is enough, as for the chosen functions `wveOLAS` is just going to route the same call to `veOLAS`.